src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -45,8 +54,11 @@ public override void OnGrab(GrabMessage message)
             var environmentVariables = new StringDictionary();
 
             environmentVariables.Add("Sonarr_EventType", "Grab");
+            environmentVariables.Add("Sonarr_InstanceName", _configFileProvider.InstanceName.ToString());
+            environmentVariables.Add("Sonarr_ApplicationUrl", _configService.ApplicationUrl.ToString());
             environmentVariables.Add("Sonarr_Series_Id", series.Id.ToString());
             environmentVariables.Add("Sonarr_Series_Title", series.Title);
+            environmentVariables.Add("Sonarr_Series_TitleSlug", series.TitleSlug.ToString());
```

Comment:
Don’t call `.ToString()` when they’re already strings, (everything you’re adding is already a string).

Reply:
Updated.